packages/deployment/upgrade-test/Dockerfile

```
@@ -69,6 +70,23 @@ RUN chmod +x ./upgrade-test-scripts/*.sh
 SHELL ["/bin/bash", "-c"]
 RUN . ./upgrade-test-scripts/start_to_to.sh
 
+ARG DEST_IMAGE
+#this is agoric-upgrade-10 upgrading to 11
+#it's a separate target because agoric-upgrade-10 takes so long to test
+FROM ghcr.io/agoric/agoric-sdk:34 as agoric-upgrade-10-to-11
```

I believe this should be 35 -- I mistakenly used 34 which is the previous RC. I can change in a separate PR or we can change here.
```suggestion
FROM ghcr.io/agoric/agoric-sdk:35 as agoric-upgrade-10-to-11
```